From 11bd2e37da2dbf9aadb37c97da18030d4100981a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lawrence=20Vel=C3=A1zquez?= <vq@larryv.me>
Date: Thu, 3 Jun 2021 23:11:53 -0400
Subject: [PATCH 2/6] Call find(1) with an explicit path in build

POSIX [*] does not specify the behavior of find(1) when its first
operand begins with '-'. GNU find(1) chooses to search the current
directory, but other implementations consider it an error. Specify '.'
explicitly for compatibility with non-GNU implementations.

This change is necessary to build on macOS.

  [*]: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/find.html
---
 icmake/icm_prepare | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/icmake/icm_prepare b/icmake/icm_prepare
index 398f55e..df3ec53 100755
--- a/icmake/icm_prepare
+++ b/icmake/icm_prepare
@@ -22,7 +22,7 @@ try()
 echo Creating the intermediate destination directory ./tmp
 echo and removing any old files
 try rm -rf tmp icmake/tmp icmun/tmp/ icm-*/tmp
-try find -name '*.o' -exec rm -f '{}' \;
+try find . -name '*.o' -exec rm -f '{}' \;
 try mkdir -p tmp
 
 echo    Writing tmp/ROOT
-- 
2.31.1

